js/notify/settings/configure-notification-position.js

```diff
@@ -27,7 +28,7 @@ let windowConfig = {
     resizable: false,
     webPreferences: {
         preload: path.join(__dirname, 'configure-notification-position-preload.js'),
-        sandbox: true,
+        sandbox: sandboxed,
```

Comment:
I would prefer this to be abstracted out. Will merge this for now to have builds ready on Jenkins. We can address this as part of refactoring.